### PR TITLE
Three career pages (ui/ux, product management and web development) html files

### PR DESCRIPTION
**Pull request by JoyChukwukere**

### CODE REVIEW

**Comment by goke-epapa** [comment]
Please update the PR title to be more descriptive and add a description to the PR

Suggestion:
> Basic career pages with images

**Review by faithakinbobola — CHANGES_REQUESTED** [review]
You only changed the image folder.  You've not renamed it in your html files

**Comment by JoyChukwukere** [comment]
OKAY

**Review by Iyiola-stack — APPROVED** [review]

File: ProductDevelopment.html [hunk 1]
```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>PRODUCT MANAGEMENT| TechHub-Career</title>
+    </head>
+    <body>
+        <h1>PRODUCT MANAGEMENT</h1>
+        <img src="./image/ProductManagement.png" alt="">
```
Comment (goke-epapa) [inline ProductDevelopment.html]: Missing an `alt` text - https://www.w3schools.com/tags/att_img_alt.asp

File: UiUx.html [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+<html lang="en">
+  <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>UI/UX | TechHub-Career</title>
+    </head>
+
+    <body>
+        <h1>UI/UX</h1>
+        <img src="./image/UiUx.png" alt="">
```
Comment (goke-epapa) [inline UiUx.html]: Missing alt text - https://www.w3schools.com/tags/att_img_alt.asp

File: WebDevelopment.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>WEB DEVELOPMENT| TechHub-Career</title>
+    </head>
+    <body>
+        <h1>WEB DEVELOPMENT</h1>
+        <img src="./image/Web Develoipment.png" alt="">
```
Comment (goke-epapa) [inline WebDevelopment.html]: Missing alt text - https://www.w3schools.com/tags/att_img_alt.asp

File: ProductDevelopment.html [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+            But before you tackle those, you’ll want to master at least some product management hard skills. These commonly include
+            research skills, an understanding of how businesses operate, and general knowledge of product management best practices.
+            Once you have a solid foundation of hard product management skills, you can start focusing on how to best improve your
+            soft skills to be a well-rounded product manager.
+        </P>
+        <p>
+            <strong>Familiarity with Economics</strong>
+        </p>
+        <b>
+            NYou don’t need a degree in economics to be a product manager (although that degree couldn’t hurt, either), but you
```
Comment (goke-epapa) [inline ProductDevelopment.html]:
```suggestion
            You don’t need a degree in economics to be a product manager (although that degree couldn’t hurt, either), but you
```

File: ProductDevelopment.html [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+            need.
+            Having a basic understanding of economics goes a long way toward understanding concepts you’ll use every day in this
+            career. These concepts could include the need for trade-offs, costs vs. benefit analyses, future consequences of current
+            actions (think technical debt!), and others.
+            </P>
+            <p>
+                <strong>Basic Product Management Knowledge</strong>
+            </p>
+            <P>
+                AProduct management is a sweeping discipline. There are many ways to approach product development, so getting started can
```
Comment (goke-epapa) [inline ProductDevelopment.html]:
```suggestion
                Product management is a sweeping discipline. There are many ways to approach product development, so getting started can
```

File: WebDevelopment.html [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+            <P>
+                Even the most minute typo can lead to a 404. Programmers learn to re-read their code multiple times as they check for
+                errors and look for efficiencies. It takes patience, laser-focus, and an ability to look at your own work with an
+                objective eye.
+            </P>
+            <H2>COURSES</H2>
+            <P>The following courses would help you in your career as UI/UX designe></P>
+            <a
+                href="https://www.pluralsight.com/browse/software-development/web-development?aid=7010a000002LUv2AAG&promo=&utm_source=non_branded&utm_medium=digital_paid_search_google&utm_campaign=XYZ_EMEA_Dynamic&utm_content=&gclid=Cj0KCQjw-af6BRC5ARIsAALPIlVbq6p47uycFayj344Jb0C5jtAspXjIh8rmKych886hOvVMqj1H-7IaAjTzEALw_wcB">Getting
+                GROW YOUR WEB DEVELOPMENT SKILLSUnderline</a>
```
Comment (goke-epapa) [inline WebDevelopment.html]:
```suggestion
                GROW YOUR WEB DEVELOPMENT SKILLS</a>
```

File: WebDevelopment.html [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+            <br>
+            <a href="https://www.udemy.com/?utm_source=adwords&utm_medium=udemyads&utm_campaign=Generic-Broad_la.EN_cc.ROW&utm_content=deal4584&utm_term=_._ag_86841142536_._ad_405920124403_._kw_learn%20online_._de_c_._dm__._pl__._ti_kwd-11046951_._li_1010294_._pd__._&matchtype=b&gclid=Cj0KCQjw-af6BRC5ARIsAALPIlXKIheYDCz6aR5iiWD7SFFoDvpdnXGPDfeXxCZ-1zNOlZrEtSDWvzAaAnSXEALw_wcB">Web Development
```
Comment (goke-epapa) [inline WebDevelopment.html]: The URL is too long and also contains info you don't need
```suggestion
            <a href="https://www.udemy.com">Web Development
```

File: WebDevelopment.html [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+                </a>
+            <br>
+            <a href="https://www.codecademy.com/learn/paths/web-development?g_acctid=243-039-7011&g_keywordid=kwd-12372542&g_adid=435749871538&g_keyword=web%20development%20courses&g_campaign=ROW+Career+Path%3A+Pro+-+Exact&g_adtype=search&g_network=g&g_adgroupid=102650142633&g_campaignid=10074200774&utm_id=t_kwd-12372542:ag_102650142633:cp_10074200774:n_g:d_c&utm_term=web%20development%20courses&utm_campaign=ROW%20Career%20Path%3A%20Pro%20-%20Exact&utm_source=google&utm_medium=paid-search&utm_content=435749871538&hsa_acc=2430397011&hsa_cam=10074200774&hsa_grp=102650142633&hsa_ad=435749871538&hsa_src=g&hsa_tgt=kwd-12372542&hsa_kw=web%20development%20courses&hsa_mt=e&hsa_net=adwords&hsa_ver=3&gclid=Cj0KCQjw-af6BRC5ARIsAALPIlW2BuGwH6qXOeh3yCwY-dvr22ltyymN-m8w1JjcFzOJcA2VMyvzZ50aAnrLEALw_wcB">Learn
```
Comment (goke-epapa) [inline WebDevelopment.html]:
```suggestion
            <a href="https://www.codecademy.com/learn/paths/web-development">Learn
```

File: WebDevelopment.html [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+            </p>
+            <P>
+                Even the most minute typo can lead to a 404. Programmers learn to re-read their code multiple times as they check for
+                errors and look for efficiencies. It takes patience, laser-focus, and an ability to look at your own work with an
+                objective eye.
+            </P>
+            <H2>COURSES</H2>
+            <P>The following courses would help you in your career as UI/UX designe></P>
+            <a
+                href="https://www.pluralsight.com/browse/software-development/web-development?aid=7010a000002LUv2AAG&promo=&utm_source=non_branded&utm_medium=digital_paid_search_google&utm_campaign=XYZ_EMEA_Dynamic&utm_content=&gclid=Cj0KCQjw-af6BRC5ARIsAALPIlVbq6p47uycFayj344Jb0C5jtAspXjIh8rmKych886hOvVMqj1H-7IaAjTzEALw_wcB">Getting
```
Comment (goke-epapa) [inline WebDevelopment.html]:
```suggestion
                href="https://www.pluralsight.com/browse/software-development/web-development">Getting
```

File: WebDevelopment.html [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+            <p>
+                <strong>Attention to detail</strong>
+            </p>
+            <P>
+                Even the most minute typo can lead to a 404. Programmers learn to re-read their code multiple times as they check for
+                errors and look for efficiencies. It takes patience, laser-focus, and an ability to look at your own work with an
+                objective eye.
+            </P>
+            <H2>COURSES</H2>
+            <P>The following courses would help you in your career as UI/UX designe></P>
```
Comment (goke-epapa) [inline WebDevelopment.html]:
```suggestion
            <p>The following courses would help you in your career as UI/UX designer</p>
```

File: WebDevelopment.html [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+        </p>
+        <b>
+            Programming languages operate on rules and definitions. If you’re naturally skilled at thinking through problems
+            logically, you’ll pick up these languages much more quickly. As you see them in action, you’ll develop a knack for
+            breaking down statements into their core parts and predicting logical outcomes.
+            </P>
+            <p>
+                <strong>Attention to detail</strong>
+            </p>
+            <P>
```
Comment (goke-epapa) [inline WebDevelopment.html]:
```suggestion
            <p>
```

File: WebDevelopment.html [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+                Even the most minute typo can lead to a 404. Programmers learn to re-read their code multiple times as they check for
+                errors and look for efficiencies. It takes patience, laser-focus, and an ability to look at your own work with an
+                objective eye.
+            </P>
```
Comment (goke-epapa) [inline WebDevelopment.html]:
```suggestion
            </p>
```

File: ProductDevelopment.html [hunk 13]
```diff
@@ -0,0 +1,146 @@
...
+                <br>
+                But we believe mastering this ability is important enough to a product manager that it deserves a place on this list of
+                hard product management skills. Particularly because you can apply science to it. Objective prioritization frameworks
+                are helpful for many prioritization decisions. And we think working knowledge of these frameworks and when and how to
+                use them is a critical product management hard skill.
+            </P>
+            <H2>COURSES</H2>
+            <P>The following courses would help you in your career as UI/UX designe></P>
+            <a
+                href="https://www.udemy.com/course/the-complete-product-management-course/?gclid=Cj0KCQjw-af6BRC5ARIsAALPIlXF4ua_SIYNOgEgfDPh7thpZ0dnGxv_1S9odlqKMw9rFMeaOi9Hl5QaAjnbEALw_wcB&matchtype=e&utm_campaign=LongTail_la.EN_cc.ROW&utm_content=deal4584&utm_medium=udemyads&utm_source=adwords&utm_term=_._ag_78754139893_._ad_388036022847_._kw_product+management+course_._de_c_._dm__._pl__._ti_kwd-300198304963_._li_1010294_._pd__._">Getting
```
Comment (goke-epapa) [inline ProductDevelopment.html]:
```suggestion
                href="https://www.udemy.com/course/the-complete-product-management-course">Getting
```

File: UiUx.html [hunk 14]
```diff
@@ -0,0 +1,151 @@
...
+                Learning some of the coding languages necessary for front-end development may be all you need. Languages
+                such as HTML,
+                CSS, Javascript and jQuery can help you understand what’s possible from a technological perspective and help
+                ease the
+                transition between design-developer handoff.
+            </p>
+            <H2>COURSES</H2>
+            <P>The following courses would help you in your career as UI/UX designe></P>
+                <a
+                    href="https://www.pluralsight.com/courses/getting-started-ux-design?aid=7010a000002LUv2AAG&promo=&utm_source=non_branded&utm_medium=digital_paid_search_google&utm_campaign=XYZ_EMEA_Dynamic&utm_content=&gclid=Cj0KCQjw-af6BRC5ARIsAALPIlXbGZmYKjMHaeD0PDFh5OWJzywTPf2DP5BpaZmAwZGgNgmvy67pl_gaAkH_EALw_wcB">Getting
```
Comment (goke-epapa) [inline UiUx.html]:
```suggestion
                    href="https://www.pluralsight.com/courses/getting-started-ux-design">Getting
```

File: UiUx.html [hunk 15]
```diff
@@ -0,0 +1,151 @@
...
+                    Started in UX Design</a>
+                <br>
+                <a
+                    href="https://www.udemy.com/course/the-complete-app-design-course-ux-and-ui-design/?utm_source=adwords&utm_medium=udemyads&utm_campaign=LongTail_la.EN_cc.ROW&utm_content=deal4584&utm_term=_._ag_77879424374_._ad_437497333851_._kw__._de_c_._dm__._pl__._ti_dsa-1007766171592_._li_1010294_._pd__._&matchtype=b&gclid=Cj0KCQjw-af6BRC5ARIsAALPIlWKV4ncAAqIzYexGUlgYJJWz-2-Kq-8vNuFmF-AsUV1gQIJX_IE4EsaAkGmEALw_wcB">The
```
Comment (goke-epapa) [inline UiUx.html]:
```suggestion
                    href="https://www.udemy.com/course/the-complete-app-design-course-ux-and-ui-design">The
```

File: UiUx.html [hunk 16]
```diff
@@ -0,0 +1,151 @@
...
+                    Complete App Design Course - UX, UI and Design Thinking</a>
+                <br>
+                <a
+                    href="https://www.interaction-design.org/courses?ad-set=ui-ux-courses&gclid=Cj0KCQjw-af6BRC5ARIsAALPIlUUAoHuqD_RENtwImobVrdsOf_r1cwDZrhJxaV7Pr8wNaJ18jdOPpMaApaZEALw_wcB">Learn
```
Comment (goke-epapa) [inline UiUx.html]:
```suggestion
                    href="https://www.interaction-design.org/courses">Learn
```

File: ProductDevelopment.html [hunk 17]
```diff
@@ -0,0 +1,146 @@
...
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>PRODUCT MANAGEMENT| TechHub-Career</title>
+    </head>
+    <body>
+        <h1>PRODUCT MANAGEMENT</h1>
+        <img src="./image/ProductManagement.png" alt="">
+        <H2>DESCRIPTION</H2>
+        <P>
```
Comment (goke-epapa) [inline ProductDevelopment.html]:
```suggestion
        <p>
```

File: ProductDevelopment.html [hunk 18]
```diff
@@ -0,0 +1,146 @@
...
+            Product management is an organisational function within a company dealing with new product development,
+            business justification, planning, verification, forecasting, pricing,
+            product launch, and marketing of a product or products at all stages of the product lifecycle.
+        </P>
```
Comment (goke-epapa) [inline ProductDevelopment.html]:
```suggestion
        </p>
```

File: ProductDevelopment.html [hunk 19]
```diff
@@ -0,0 +1,146 @@
...
+        <H2>SKILLS</H2>
```
Comment (goke-epapa) [inline ProductDevelopment.html]:
```suggestion
        <h2>SKILLS</h2>
```

File: ProductDevelopment.html [hunk 20]
```diff
@@ -0,0 +1,146 @@
...
+                <strong>Proficiency in Research and Analysis</strong>
+            </p>
+            <P>
+                Pivotal’s Director of Marketing Ronan Dunlop, put it this way: “Data is vital to a product manager. All forms of data.”
+                As we’ve pointed out here many times, your product roadmap decisions should be data-driven, not based entirely on your
+                intuition (no matter how well-honed that intuition is). Data can help alert product managers to opportunities or threats
+                in the market. It can guide them to the right places to focus development resources, and it can even help them validate
+                product and feature ideas before allocating resources into them.
+                But all of this assumes that the product manager in question knows:
+                <Ul>
```
Comment (goke-epapa) [inline ProductDevelopment.html]:
```suggestion
                <ul>
```

File: ProductDevelopment.html [hunk 21]
```diff
@@ -0,0 +1,146 @@
...
+                    <li>How to conduct market research</li>
+                    <li>How to use (or build) tools to compile and analyze data</li>
+                    <li>And, how to interpret all of the data these tools generate</li>
+                </Ul>
```
Comment (goke-epapa) [inline ProductDevelopment.html]:
```suggestion
                </ul>
```

File: ProductDevelopment.html [hunk 22]
```diff
@@ -0,0 +1,146 @@
...
+                “Data can catch you off-guard. You might do your research and think, ‘Now I have all of my data points; I’ve told my
+                story and made my business case.’ But then if you present it to someone with more of an understanding of data than you
+                have, they might be able to easily poke holes in your plan.”
+                So we, like Kevin, believe that gaining proficiency in metrics, analytics, and research definitely represents a hard
+                skill every product manager should develop.
+            </P>
+            <p>
+                <strong>Ruthless (Objective) Prioritization Skills</strong>
+            </p>
+            <P>
```
Comment (goke-epapa) [inline ProductDevelopment.html]:
```suggestion
            <p>
```

File: ProductDevelopment.html [hunk 23]
```diff
@@ -0,0 +1,146 @@
...
+                support, even our stakeholders.”
+                Knowing how to prioritize, and how to respectfully turn down requests that could upset your product’s strategic
+                priorities, is an extremely valuable skill. And yes, it probably covers a grey area between hard and soft skills. The
+                webinar panel itself was torn as to how to categorize it.
+                <br>
+                But we believe mastering this ability is important enough to a product manager that it deserves a place on this list of
+                hard product management skills. Particularly because you can apply science to it. Objective prioritization frameworks
+                are helpful for many prioritization decisions. And we think working knowledge of these frameworks and when and how to
+                use them is a critical product management hard skill.
+            </P>
```
Comment (goke-epapa) [inline ProductDevelopment.html]:
```suggestion
            </p>
```

File: ProductDevelopment.html [hunk 24]
```diff
@@ -0,0 +1,146 @@
...
+            <H2>COURSES</H2>
```
Comment (goke-epapa) [inline ProductDevelopment.html]:
```suggestion
            <h2>COURSES</h2>
```

File: ProductDevelopment.html [hunk 25]
```diff
@@ -0,0 +1,146 @@
...
+            <P>The following courses would help you in your career as UI/UX designe></P>
```
Comment (goke-epapa) [inline ProductDevelopment.html]:
```suggestion
            <p>The following courses would help you in your career as UI/UX designe></p>
```

**Review by goke-epapa — CHANGES_REQUESTED** [review]
Please review your HTML tags for all files and confirm you are using lowercase and not uppercase for any of them.

File: ProductDevelopment.html (hunk 4 above)
Comment (JoyChukwukere) [reply ProductDevelopment.html]: okay

File: ProductDevelopment.html (hunk 25 above)
Comment (JoyChukwukere) [reply ProductDevelopment.html]: Please what is wrong with this tag?

Comment (goke-epapa) [reply ProductDevelopment.html]: it's in uppercase instead of lower case

File: ProductDevelopment.html (hunk 23 above)
Comment (JoyChukwukere) [reply ProductDevelopment.html]: please what do you want me to do here?

**Review by faithakinbobola — APPROVED** [review]

**Review by goke-epapa — APPROVED** [review]

**Pull request merged**